ui/component/event-inline.component.ts

```diff
@@ -1,21 +1,21 @@
-import { GetEventDto } from "../../server/events/events.dto"
-import { Column, Heading2, Heading4, Link, Paragraph, Row, RowSpaced, Span, Text } from "../core/html.elements"
-import { Component } from "../core/html.interfaces"
+import { EventModel } from "../../server/events/event.model";
+import { Link, RowSpaced, Text } from "../core/html.elements";
+import { Component } from "../core/html.interfaces";
 
-export const EventInline: Component<GetEventDto> = (event: GetEventDto) => {
+export const EventInline: Component<EventModel> = (event: EventModel) => {
 
-    const icon = event.ratingPercentage >= 50 ? "✔️": "❌";
+    const icon = event.ratingPercentage >= 50 ? "✔️" : "❌";
```

Comment:
Also, is `event.ratingPercentage >= 50` business logic of an EventModel (see comment below)?
Should we have a field/method `isRecommended` inside the model instead?
(also for https://github.com/mbrizic/ratethegame/blob/f8b046669120fca63a60deb31bf9cf56bf83623f/ui/component/event-rating.component.ts#L7)

Reply:
Definitely, good catch!